src/polyswarm_api/__init__.py

```diff
@@ -310,67 +313,84 @@ def _fix_result(self, result):
         logger.warning('Failed to get results for uuid %s in time.', uuid)
         return {'files': result['files'], 'uuid': uuid}
 
-    async def scan_url(self, to_scan, name='url'):
+    async def scan_fileobjs(self, file_objs, file_names, artifact_type):
```

Comment:
Should this instead be a list of `(file_obj, file_name)` tuples? If not, we should verify at minimum that these lists are the same length.

Reply:
Using a list of tuples now :+1: